webpack.config.js

```diff
@@ -37,6 +41,17 @@ module.exports = {
       { from: 'partials/*', to: '.' },
       { from: 'img/*', to: '.' },
     ]),
+    new ReplaceInFileWebpackPlugin([{
+      dir: 'dist',
+      files: ['plugin.json'],
+      rules: [{
+        search: '%VERSION%',
+        replace: packageJson.version
+      },{
+        search: '%TODAY%',
+        replace: moment().format('YYYY.MM.DD')
```

Comment:
Please use ISO8601 here, and refrain from using `moment`, we might replace it soon.